### ISSUE TITLE
Failed update

### ISSUE THREAD

**Issue by bfraase** [nmelhado/league-page#183]
Hey- this was totally my fault I think! But- my website said it had an update available. So I went to the instructions and clicked 'sync fork' and 'update branch.'

Unfortunately that wiped out almost everything I had done, most notably the blog and manager pages. I went back in and had to enter my sleeper league ID and got back matchups, rosters, and all that stuff etc. Do you have any idea what I might have done wrong or how I can undo my update? I don't want to go back and have to update all of the manager pages unless I really have to.

www.leagueofmisfitboys.com

<img width="338" alt="Screen Shot 2022-12-23 at 2 26 37 PM" src="https://user-images.githubusercontent.com/96595807/209406993-40a0b0db-50a2-45f1-bca7-d5d961e726e8.png">

**Comment by nmelhado** [comment]
You should be able to click on commits and look back at a previous commit. If you find one with your old work in constitution, you can copy the contents and then go back to your current version and paste them in.

If that doesn't make sense, let me know and I'll try to explain it better.

**Comment by Justatix** [comment]
hey bfraase, sorry for popping up in this issue, but out of curiosity, how were you able to change the main league picture/icon at the top of your page? I would  also like to have an individual picture for it... was not able to find a description for it.

**Comment by nmelhado** [comment]
If you want to replace the league page logo with your league's own logo, replace static/badge.png with your own 120px x 120px png file (using the same name)

**Comment by Justatix** [comment]
> If you want to replace the league page logo with your league's own logo, replace static/badge.png with your own 120px x 120px png file (using the same name)

thanks for your fast feedback, it worked properly. Do I have to follow same steps to replace favicons under static/favicons? because if so, i tried to replace them, but unfortunately, the old icons are still being shown on my browser.

**Comment by nmelhado** [comment]
@Justatix Yes, same process (making sure to match dimensions). It is most likely cached though if you did it correctly. Try opening your page in an incognito browser to check

**Comment by nmelhado** [comment]
If you click the drop-down arrow do you see any other options?

**Comment by nmelhado** [comment]
Hopefully there's a `fetch upstream` option

**Comment by bfraase** [comment]
Hey-

I appreciate you taking the time to help out with this. I have 'sync for' instead of 'fetch upstream.' Any ideas? I might just scratch it and start over again! It's a fun, although time consuming, learning process doing it over


<img width="931" alt="contribue" src="https://user-images.githubusercontent.com/96595807/233806253-f9507080-f11a-4c0b-ac58-412b7aa94941.png">

<img width="937" alt="sync fork" src="https://user-images.githubusercontent.com/96595807/233806261-420f9b17-f5f1-4ac4-a4bd-7943568cf5c0.png">
